server/game/Interfaces.ts

```diff
@@ -67,19 +68,26 @@ export interface IAbilityProps<Context> {
     cardName?: string;
 
     /**
-     * Indicates whether the ability should allow the player to trigger an attack from a unit.
-     * Can either be an {@link IInitiateAttack} property object or a function that creates one from
+     * Indicates if triggering the ability is optional (in which case the player will be offered the
+     * 'Pass' button on resolution) or if it is mandatory
+     */
+    optional?: boolean;
+
+    /**
+     * Indicates that an attack should be triggered from a friendly unit.
+     * Shorthand for `AbilityHelper.immediateEffects.attack(AttackSelectionMode.SelectAttackerAndTarget)`.
+     * Can either be an {@link IInitiateUnitAttackProperties} property object or a function that creates one from
      * an {@link AbilityContext}.
      */
-    initiateAttack?: IInitiateAttack | ((context: AbilityContext) => IInitiateAttack);
+    initiateAttack?: IInitiateUnitAttackProperties | ((context: AbilityContext) => IInitiateUnitAttackProperties);
```

Comment:
I think this could just be IInitiateAttackProperties. Unit seems implied

Reply:
Actually I guess this needs to match the system name as discussed below, since they're paired